tests/integration/cli/test_test.py

```diff
@@ -203,7 +203,7 @@ def test_uncaught_txn_err(setup_pytester, integ_project, pytester, eth_tester_pr
 def test_show_internal(setup_pytester, integ_project, pytester, eth_tester_provider):
     _ = eth_tester_provider  # Ensure using EthTester for this test.
     setup_pytester(integ_project)
-    result = pytester.runpytest_subprocess("--show-internal")
+    result = pytester.runpytest_subprocess("-p", "no:pytest_ethereum", "--show-internal")
```

Comment:
@fubuloubu this is the annoying part, have to make sure `pytest_ethereum` is disabled :( 
*thinking*

i tried disabling in the plugin but the problem happens earlier than the loading of our plugin. `pytest_ethereum` loads up and fails before ape's pytest plugin does.